docs/source/extension/extension_tutorial.rst

```diff
@@ -234,7 +234,8 @@ Fire up your favorite text editor and open the ``src/index.ts`` file in your
 extension project. Change the import at the top of the file to get a reference
 to the command palette interface and the `JupyterFrontEnd` instance.
```

Comment:
We should use double quote here to have code styling:

```suggestion
to the command palette interface and the ``JupyterFrontEnd`` instance.
```

Reply:
Just fixed/pushed this.